spec/models/user_spec.rb

```diff
@@ -46,4 +46,10 @@
     subject.login = nil
```

Comment:
would it be possible to add some `feature`/`system` tests using capybara and chrome?

https://guides.rubyonrails.org/testing.html#system-testing